src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int check_outdated_data(int maxTimeDiff, char *receivingData) {
+
+    time_t curTime; 
+
+    struct tm timeptr;
+
+    curTime = time(NULL);
+   /*  printf("Current timestamp = %ld\n", curTime); */
+
+    /* We are now converting the recieving data(Charachter string) into a timestamp via 
+    this function strptime and this timeptr struct. */
+
+    strptime(receivingData, "%Y%m%d%H%M%S", &timeptr);
+    time_t timetostamp = mktime(&timeptr);
```

Comment:
this variable should be called `timeToStamp` instead

Reply:
yes will do